test/bin/pyutils/build_bootc_images.py

```diff
@@ -228,7 +228,16 @@ def process_containerfile(groupdir, containerfile, dry_run):
     # Run template command on the input file
     cf_outfile = os.path.join(BOOTC_IMAGE_DIR, containerfile)
     run_template_cmd(cf_path, cf_outfile, dry_run)
-    if os.path.getsize(cf_outfile) == 0:
+    # Templating may generate an empty file.
+    # Detect empty files by counting uncommented lines.
+    grep_args = [
+        "grep", "--count", "--invert-match",
+        "-e", "^\\s*$", "-e", "^\\s*#",
+        cf_outfile, "||", "true"
+    ]
+    nlines = common.run_command_in_shell(grep_args, dry_run)
```

Comment:
I'm not a fan of this... Feels like we're doing a disservice for python by turning it into bash 2.0